migrations/acs.sql

```diff
@@ -23,5 +23,10 @@ SELECT
     labs_geoid as geoid,
     pff_variable as variable,
     'Y'||(:TABLE_NAME-4)::TEXT||'-'||:'TABLE_NAME' as dataset,
-    domain, c, e, m, p, z
+    domain as profile,
```

Comment:
We should just rename these columns in the tables as well instead of aliasing like this

Reply:
That's what this does? Unless you mean changing the column names in the csv's that (I think) EDM maintains.

Reply:
oh oops, sorry thought this was a query. sorry nevermind